tests/driver_at86rf215/Makefile

```diff
@@ -0,0 +1,27 @@
+include ../Makefile.tests_common
```

Comment:
Maybe add `openmote-b` as the default `BOARD`? Otherwise the test application woks for me!

```
> ifconfig
2020-03-19 11:24:24,701 #  ifconfig
2020-03-19 11:24:24,703 # Iface  5  HWaddr: 2B:CB  Channel: 26  Page: 0  NID: 0x23
2020-03-19 11:24:24,704 #           Long HWaddr: 22:68:31:23:8C:96:96:34 
2020-03-19 11:24:24,718 #            TX-Power: 0dBm  State: IDLE  max. Retrans.: 3  CSMA Retries: 4 
2020-03-19 11:24:24,719 #           AUTOACK  ACK_REQ  CSMA  L2-PDU:102 Source address length: 2
2020-03-19 11:24:24,720 #           
2020-03-19 11:24:24,733 # Iface  4  HWaddr: 2B:C9  Channel: 0  Page: 2  NID: 0x23
2020-03-19 11:24:24,733 #           Long HWaddr: 22:68:31:23:8C:96:96:36 
2020-03-19 11:24:24,734 #            TX-Power: 0dBm  State: IDLE  max. Retrans.: 3  CSMA Retries: 4 
2020-03-19 11:24:24,749 #           AUTOACK  ACK_REQ  CSMA  L2-PDU:102 Source address length: 2
2020-03-19 11:24:24,750 #           
> ifconfig channel
2020-03-19 11:24:36,684 #  ifconfig channel
2020-03-19 11:24:36,685 # error: invalid interface given
> 2020-03-19 11:25:55,367 #  PKTDUMP: data received:
2020-03-19 11:25:55,384 # ~~ SNIP  0 - size:   2 byte, type: NETTYPE_UNDEF (0)
2020-03-19 11:25:55,384 # 00000000  68  69
2020-03-19 11:25:55,385 # ~~ SNIP  1 - size:  18 byte, type: NETTYPE_NETIF (-1)
2020-03-19 11:25:55,385 # if_pid: 5  rssi: -39  lqi: 0
2020-03-19 11:25:55,386 # flags: 0x0
2020-03-19 11:25:55,386 # src_l2addr: 3F:2D
2020-03-19 11:25:55,399 # dst_l2addr: 22:68:31:23:8C:96:96:34
2020-03-19 11:25:55,400 # ~~ PKT    -  2 snips, total size:  20 byte
2020-03-19 11:25:56,919 # PKTDUMP: data received:
2020-03-19 11:25:56,935 # ~~ SNIP  0 - size:   2 byte, type: NETTYPE_UNDEF (0)
2020-03-19 11:25:56,936 # 00000000  68  69
2020-03-19 11:25:56,936 # ~~ SNIP  1 - size:  18 byte, type: NETTYPE_NETIF (-1)
2020-03-19 11:25:56,936 # if_pid: 5  rssi: -39  lqi: 0
2020-03-19 11:25:56,937 # flags: 0x0
2020-03-19 11:25:56,937 # src_l2addr: 3F:2D
2020-03-19 11:25:56,951 # dst_l2addr: 22:68:31:23:8C:96:96:34
2020-03-19 11:25:56,952 # ~~ PKT    -  2 snips, total size:  20 byte
```